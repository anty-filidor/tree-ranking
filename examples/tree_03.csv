id;parent_id;color;value;​
1;;yellow;90;​
2;1;blue;10;​
3;1;green;10;​
4;1;orange;35;​
5;1;purple;70;​
6;1;black;40;​
7;6;red;20;
8;6;red;51;
