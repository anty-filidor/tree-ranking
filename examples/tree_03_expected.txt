python color_ranking.py --file /path/tree_03.csv --criteria subtree-count​
[('red', 2), ('black', 1), ('blue', 1), ('green', 1), ('orange', 1), ('purple', 1), ('yellow', 1)]

​
python color_ranking.py --file /path/tree_03.csv --criteria subtree-maxdepth​
[('black', 1), ('blue', 1), ('green', 1), ('orange', 1), ('purple', 1), ('red', 1), ('yellow', 1)]

​
python color_ranking.py --file /path/tree_03.csv --criteria subtree-average-value​
[('yellow', 90), ('purple', 70), ('black', 40), ('orange', 35), ('red', 35), ('blue', 10), ('green', 10)]




